### PR TITLE
Updated GET /projects/quote to include document-leverage.

### PR DESCRIPTION
**Pull request by judithnursalim**
Fixes https://github.com/lilt/front/issues/7143

## Engineering Notes

- Regenerated client libraries to expose `project.numSourceTokens` and `document/numSourceTokens` in GET /projects/quote.

### CODE REVIEW

File: docs/DocumentsApi.md [hunk 1]
```diff
@@ -25,12 +25,12 @@ Assign and unassign a Document for translation and/or review.
...
 ```java
 // Import classes:
-import com.lilt.ApiClient;
-import com.lilt.ApiException;
-import com.lilt.Configuration;
-import com.lilt.auth.*;
-import com.lilt.models.*;
-import com.lilt.api.DocumentsApi;
+import org.openapitools.client.ApiClient;
+import org.openapitools.client.ApiException;
```
Comment (connorguy) [inline docs/DocumentsApi.md]: You will need to tweak the generator call to make this match up with the Lilt lib name.

**Review by connorguy — CHANGES_REQUESTED** [review]

File: docs/DocumentsApi.md (hunk 1 above)
Comment (connorguy) [reply docs/DocumentsApi.md]: This is what I used
```
openapi-generator generate \
 -i /Users/connorguy/lilt/repos/front_end/front/doc/public-http-api-swagger.yaml \
 -g java \
 --git-host github.com \
 --git-user-id lilt \
 --package-name lilt \
 -o /Users/connorguy/Desktop/api_work/lilt-java \
 -c /Users/connorguy/Desktop/api_work/config.json
```
Also double check that commit https://github.com/lilt/lilt-java/commit/0dbd1d16612cfc528c03b828906bc2681436779b and https://github.com/lilt/lilt-java/commit/777599ffe8749832ee604beaa34a8d065d8028f7 are not reverted, I had to manually correct these.

Comment (tgallant) [reply docs/DocumentsApi.md]: > -c /Users/connorguy/Desktop/api_work/config.json

what's in this config file?

Comment (judithnursalim) [reply docs/DocumentsApi.md]: From Connor

```
{
    "modelPackage" : "com.lilt.model",
    "apiPackage" : "com.lilt.api"
}
```

Comment (judithnursalim) [reply docs/DocumentsApi.md]: > You will need to tweak the generator call to make this match up with the Lilt lib name.

- [x] Used config file to override `modelPackage` and `apiPackage`.

> Also double check that commit 0dbd1d1 and 777599f are not reverted, I had to manually correct these.

- [x] They weren't reverted.

**Review by connorguy — APPROVED** [review]
lgtm

**Pull request merged**